lstchain/io/lsteventsource.py

```diff
@@ -0,0 +1,379 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+EventSource for LSTCam protobuf-fits.fz-files.
+
+Needs protozfits v1.4.2 from github.com/cta-sst-1m/protozfitsreader
+"""
+import glob
+import numpy as np
+from astropy import units as u
+import struct
+
+from ctapipe.core import Provenance
+from ctapipe.instrument import TelescopeDescription, SubarrayDescription, \
+    CameraGeometry, OpticsDescription
+from ctapipe.io.eventsource import EventSource
+from .containers import LSTDataContainer
+
+
+__all__ = ['LSTEventSource']
+
+
+class LSTEventSource(EventSource):
```

Comment:
It would be nice to have a small test file to unit test `LSTEventSource`.

Reply:
This test is already in ctapipe, no problem to add it in lstchain, where should I put it?
Should we create a test subdirectory?

Reply:
Are you therefor adding this piece of code here for future developments?
Otherwise better just call the ctapipe version which is tested no?

If the test subdirectory is missing, you can create it yes.

Reply:
Yes, the problem of ctapipe is that PR acceptance takes too much time also they want to take specific telescope readers out of ctapipe and have them as plugin. I am actually waiting for Dominik to indicate how to proceed. In the meanwhile, I would like to share in LST this new version of the reader

Reply:
Ok great.
In this case, a unit test would be welcome.
Thanks.